### PR TITLE
Author suitability extension

### PR DESCRIPTION
**Pull request by duncandewhurst**

### CODE REVIEW

File: release-schema.json [hunk 1]
```diff
@@ -1 +1,47 @@
-{}
+{
+  "definitions": {
+    "Tender": {
+      "properties": {
+        "suitability": {
+          "title": "Suitability",
+          "$ref": "#/definitions/Suitability",
+          "description": "Information the suitability of the tender for different types of tenderer."
```
Comment (odscjen) [inline release-schema.json]:
```suggestion
          "description": "Information about the suitability of the tender for different types of tenderer."
```

File: release-schema.json [hunk 2]
```diff
@@ -1 +1,47 @@
...
+        }
+      }
+    },
+    "Lot": {
+      "properties": {
+        "suitability": {
+          "title": "Suitability",
+          "$ref": "#/definitions/Suitability",
+          "description": "Information the suitability of the lot for different types of tenderer."
```
Comment (odscjen) [inline release-schema.json]:
```suggestion
          "description": "Information about the suitability of the lot for different types of tenderer."
```

File: release-schema.json [hunk 3]
```diff
@@ -1 +1,47 @@
...
+        }
+      }
+    },
+    "LotGroup": {
+      "properties": {
+        "suitability": {
+          "title": "Suitability",
+          "$ref": "#/definitions/Suitability",
+          "description": "Information the suitability of the lot group for different types of tenderer."
```
Comment (odscjen) [inline release-schema.json]:
```suggestion
          "description": "Information about the suitability of the lot group for different types of tenderer."
```

**Review by odscjen — CHANGES_REQUESTED** [review]
I'm assuming that even though we're only including 'sme' in this extension the wording in both the extension description and the description of `Suitability` is broad in order to allow for the inclusion of other types of tenderer in the future or via local extensions?

**Comment by duncandewhurst** [comment]
@odscjen, yep, exactly.

File: README.md [hunk 4]
```diff
@@ -1,180 +1,64 @@
-# OCDS Extension Template Repository
+# Suitability
 
-[OCDS extensions](https://standard.open-contracting.org/latest/en/extensions/) can be used to declare additional fields, objects and sections that will be included in an OCDS file, but which are not in the core schema, and to add or extend codelists. When any OCDS publication cites an extension in its' package metadata, it should be validated against the schema with that extension applied. 
+Adds a suitability object to tenders, lots and lot groups to describe suitability for different types of tenderer.
```
Comment (jpmckinney) [inline README.md]:
```suggestion
Adds a suitability object to the tender, lot and lot group objects to describe their suitability to different types of tenderers.
```

File: README.md [hunk 5]
```diff
@@ -1,180 +1,64 @@
...
 
-This repository provides a template of an extension that people can base their own on.
+If you are using the [Lots extension](https://extensions.open-contracting.org/en/extensions/lots/master/), [follow its guidance](https://extensions.open-contracting.org/en/extensions/lots/master/#usage) on whether to use `tender.lots` fields or `tender` fields.
```
Comment (jpmckinney) [inline README.md:7]:
```suggestion
# Guidance

If you are using the [Lots extension](https://extensions.open-contracting.org/en/extensions/lots/master/), [follow its guidance](https://extensions.open-contracting.org/en/extensions/lots/master/#usage) on whether to use `tender.lots` fields or `tender` fields.
```

File: README.md [hunk 6]
```diff
@@ -1,180 +1,64 @@
...
 
-## Getting started
+## Legal context
 
-To create an extension based on this template, you should:
+In the European Union, this extension's fields correspond to [eForms BT-726 (Suitable For SMEs)](https://github.com/eForms/eForms). For correspondences to eForms fields, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/).
```
Comment (jpmckinney) [inline README.md]:
```suggestion
In the European Union, this extension's fields correspond to [eForms BT-726 (Suitable For SMEs)](https://github.com/eForms/eForms). For correspondences to eForms fields, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/).
```

File: extension.json [hunk 7]
```diff
@@ -1,30 +1,24 @@
 {
   "name": {
-    "en": "Name of extension (required in at least one language)"
+    "en": "Suitability"
   },
   "description": {
-    "en": "A brief description of what this extension does, this will be pulled into the main docs as a link (required in at least one language)"
+    "en": " Adds a suitability object to tenders, lots and lot groups."
```
Comment (jpmckinney) [inline extension.json]:
```suggestion
    "en": "Adds a suitability object to the tender, lot and lot group objects to describe their suitability to different types of tenderers."
```

File: release-schema.json [hunk 8]
```diff
@@ -1 +1,47 @@
-{}
+{
+  "definitions": {
+    "Tender": {
+      "properties": {
+        "suitability": {
+          "title": "Suitability",
+          "description": "Information about the suitability of the tender for different types of tenderer.",
```
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
          "description": "Information about the suitability of the contracting process to different types of tenderers.",
```

File: release-schema.json [hunk 9]
```diff
@@ -1 +1,47 @@
...
+          "$ref": "#/definitions/Suitability"
+        }
+      }
+    },
+    "Lot": {
+      "properties": {
+        "suitability": {
+          "title": "Suitability",
+          "description": "Information about the suitability of the lot for different types of tenderer.",
```
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
          "description": "Information about the suitability of the lot to different types of tenderers.",
```

File: release-schema.json [hunk 10]
```diff
@@ -1 +1,47 @@
...
+          "$ref": "#/definitions/Suitability"
+        }
+      }
+    },
+    "LotGroup": {
+      "properties": {
+        "suitability": {
+          "title": "Suitability",
+          "description": "Information about the suitability of the lot group for different types of tenderer.",
```
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
          "description": "Information about the suitability of the lot group to different types of tenderers.",
```

File: release-schema.json [hunk 11]
```diff
@@ -1 +1,47 @@
...
+          "$ref": "#/definitions/Suitability"
+        }
+      }
+    },
+    "Suitability": {
+      "title": "Suitability",
+      "description": "Information about the suitability of tenders, lots and lot groups for different types of tenderer.",
```
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
      "description": "Information about the suitability of a contracting process, lot or lot group to different types of tenderers.",
```

File: release-schema.json [hunk 12]
```diff
@@ -1 +1,47 @@
...
+      "type": "object",
+      "properties": {
+        "sme": {
+          "title": "Small and medium enterprises",
+          "description": "Whether the tender, lot or lot group is suitable for small and medium enterprises.",
```
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
          "description": "Whether the contracting process, lot or lot group is suitable to small and medium enterprises.",
```

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**